.circleci/config.yml

```diff
@@ -43,31 +46,35 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - test-<< parameters.python_version >>-{{ checksum "requirements/test.txt" }}
+          - test-tox-py<< parameters.python_version >>-<< parameters.django_version >>-{{ checksum "requirements/test.txt" }}
           # fallback to using the latest cache if no exact match is found
-          - test-<< parameters.python_version >>-
+          - test-tox-py<< parameters.python_version >>-<< parameters.django_version >>-
 
       - run:
           name: activate virtualenv and install dependencies.
           command: |
-            test -d venv || virtualenv ./venv
+            virtualenv ./venv
             . venv/bin/activate
-            make test_requirements
-
-      - save_cache:
-          paths:
-            - ./venv
-          key: test-<< parameters.python_version >>-{{ checksum "requirements/test.txt" }}
+            pip install -r requirements/tox.txt
 
       - run:
           name: Run tests
           command: |
             . venv/bin/activate
-            coverage run --source="." manage.py test
-            coverage report --fail-under=71 -m
-            pycodestyle ./eox_tagging
-            pylint ./eox_tagging --rcfile=./setup.cfg
-            isort --check-only --recursive --diff ./eox_tagging
+            tox -e ${TOX_ENV//./} -- coverage run --source="." manage.py test
```

Comment:
why not make run-test ??? @morenol

Reply:
It was not being used before. But now I added that too.